packages/react/src/button/Button.css.ts

```diff
@@ -80,22 +79,28 @@ export const button = recipe({
         },
       }),
     },
+    icon: {
+      false: {},
+      true: {},
+    },
     size: {
-      sm: {
+      sm: style({
```

Comment:
we can't use `style` and mix sprinkle props here - i'm not sure we need `style` here at all since everything should be covered under sprinkles

Reply:
addressed this bhai